ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rhcs.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
```

Comment:
May be put it under terraform-provider folder? https://github.com/openshift/release/tree/5d1f9c75140f63481edbb79bbb897c1a27c61d22/ci-operator/jobs/terraform-redhat

Reply:
Put here for future report portal . We can move it in future after I figure out how to re-use report portal